tests/jquery.flot.hover.Test.js

```diff
@@ -259,10 +259,14 @@ describe("flot hover plugin", function () {
             expect(getEntireCanvasData(canvas)).toContainPixelColor(rgba(10, 20, 30, 1));
         });
 
-        it('should call hooked function when canvas is hovered over', function() {
+        it('should call hooked function when canvas is hovered over', function(done) {
             var spy = jasmine.createSpy('spy');
+            var hookCallback = function() {
+                spy();
```

Comment:
Why is the spy needed?

Reply:
I believe because this is what needed to assert against. I don't believe I could pass in 'hookCallback' to the expect call.

Reply:
I don't think you need an assert at all. The test will timeout and fail if the hook callback doesn't get called.